salt/states/pkg.py

```diff
@@ -2964,7 +2964,7 @@ def _uninstall(
 
     try:
         pkg_params = __salt__["pkg_resource.parse_targets"](
-            name, pkgs, normalize=normalize
+            name, pkgs, normalize=normalize, version=version
```

Comment:
Looks like this is fixed here: https://github.com/saltstack/salt/blob/3006.x/salt/states/pkg.py#L2887